django/api/viewsets/upload.py

```diff
@@ -49,10 +51,10 @@ def datasets_list(self, request):
     @method_decorator(check_upload_permission())
     def import_data(self, request):
         filename = request.data.get("filename")
-        dataset_selected = request.data.get("dataset_selected")
-        replace_data = request.data.get("replace_data", False)
+        dataset_selected = request.data.get("datasetSelected")
```

Comment:
I was trying it locally and it seems like these changes are giving me errors (they should be snake_cased); perhaps when you were resolving conflicts you picked the old changes, not the current ones we see in the 0.4.0 release branch?